config/environment.rb

```diff
@@ -1,5 +1,17 @@
 # Load the Rails application.
 require_relative 'application'
 
+
+ActionMailer::Base.smtp_settings = {
+  :user_name => 'gbrlmrllo',
+  :password => 'easyrent2017',
```

Comment:
We must also use environment variables for some data, we can not make [hard-code](https://es.wikipedia.org/wiki/Hard_code), for security reasons and good practices.